### DISCUSSION TITLE
Need help to get Started

### DISCUSSION THREAD

**Discussion by danishhansari** [vercel/next.js#44216] [Help]
Dear  Maintainer,

I am writing to express my interest in getting involved in your community and contributing to your project. I am a passionate developer with a strong interest in open source software and I believe that your project has the potential to make a real impact.

I would like to know more about how I can get involved and start contributing to your project. Are there any specific areas that you are currently looking for help in? Do you have any resources or guides that can help me get started as a contributor?

I am excited about the opportunity to work with your team and contribute to this important project. Please let me know if there is anything I can do to get started.

Thank you for considering my request

**Comment by ciruz** [comment]
Checkout the Contributing section https://github.com/vercel/next.js#contributing and the contribution guide https://github.com/vercel/next.js/blob/canary/contributing.md. There are also many issues labeled as ``good first issue``, check them out here: https://github.com/vercel/next.js/labels/good%20first%20issue
